packages/dd-trace/src/platform/node/metrics.js

```diff
@@ -79,6 +80,8 @@ module.exports = function () {
         }, INTERVAL)
       }
 
+      metrics.boolean('datadog.tracer.node.metrics.native.enabled', !!nativeMetrics)
```

Comment:
this doesn't seem super useful, easy enough to just see if they have certain metrics reporting to their account.